tsconfig.json

```diff
@@ -6,7 +6,7 @@
         // Import non-ES modules as default imports.
         "esModuleInterop": true,
         // Don't transpile jsx (handled by babel w/ emotion)
-        "jsx": "preserve",
+        "jsx": "react",
```

Comment:
What does this mean?

Reply:
This is to get it to work with `ts-jest`. Client and server configs still use `preserve`